src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -112,6 +112,12 @@ foam.CLASS({
         // for it which doesn't jive with the AbstractFObjectPropertyInfo.
         return { class: 'foam.u2.borders.NullBorder' };
       }
+    },
+    {
+      class: 'Boolean',
```

Comment:
Why would we not make this permission based?

Reply:
We could, but not all DAOs/models use permissions for authorization.

Reply:
Are we planning to copy all the comics/DAOController enableFoo properties?

Reply:
We could. I wasn't planning on doing so until I had a concrete need though.